tests/phpunit/api/v3/ContactTest.php

```diff
@@ -1721,17 +1726,15 @@ public function testMergeOrganizations() {
     $membershipParams = [
       'membership_type_id' => $membershipType["id"],
       'contact_id' => $organizationID1,
-      'start_date' => "01/01/2015",
-      'join_date' => "01/01/2010",
-      'end_date' => "12/31/2015",
+      'start_date' => '01/01/2015',
+      'join_date' => '01/01/2010',
+      'end_date' => '12/31/2015',
     ];
-    $ownermembershipid = $this->contactMembershipCreate($membershipParams);
+    $ownerMembershipID = $this->contactMembershipCreate($membershipParams);
 
-    $contactmembership = $this->callAPISuccess("membership", "getsingle", [
-      "contact_id" => $contact["id"],
-    ]);
+    $contactmembership = $this->callAPISuccessGetSingle('membership', ['contact_id' => $contact["id"],]);
```

Comment:
minor thing but why the trailing `,? @eileenmcnaughton `

Reply:
Jenkins didn't think it was minor